tests/test_models.py

```diff
@@ -554,16 +554,61 @@ def test_service_regions(self, cls_name):
             .format(cls_name, 'ServiceRegionPolygon')
         assert err_msg == str(excinfo.value)
 
+    def test_cost_per_hour(self, cls_name):
```

Comment:
I know this follows the pattern marked in the rest of the tests, but I think the name of these tests should be stating the fact that we are expecting a validation error (I was expecting a positive test case when I first read the test name).

Reply:
Suffixed "validation_fail" to each test